Northstar.CustomServers/mod/scripts/vscripts/faction_xp.gnut

```diff
@@ -3,8 +3,11 @@ global function AddFactionXP
 void function AddFactionXP( entity player, int amount )
 {
 	string faction = GetFactionChoice( player )
+	int oldLevel = FactionGetLevel( player, faction )
 	// increment xp
 	player.SetPersistentVar( "factionXP[" + faction + "]", min( FactionGetXP( player, faction ) + amount, FactionGetMaxXP( faction ) ) )
 	
 	// note: no notif for faction level up
+	if ( FactionGetLevel( player, faction ) != oldLevel )
```

Comment:
```suggestion
	if ( FactionGetLevel( player, faction ) > oldLevel )
```
could it happen that the level gets reduced by some bug or server fucking around?

Reply:
Idk, this was mostly copied code from similar files